src/ASP/system/sql/data.sql

```diff
@@ -231,6 +231,50 @@ INSERT INTO `game_mod`(`id`, `name`, `longname`, `authorized`) VALUES (2, 'xpack
 INSERT INTO `game_mod`(`id`, `name`, `longname`, `authorized`) VALUES (3, 'aix2', 'Allied Intent Extended 2.0', 0);
 INSERT INTO `game_mod`(`id`, `name`, `longname`, `authorized`) VALUES (4, 'naw', 'Nations At War', 0);
 
+--
+-- Dumping data for table `map`
+--
+
+-- Middle Eastern Theatre
+INSERT INTO `map`(`id`, `name`, `displayname`) VALUES (0, 'kubra_dam', 'Kubra Dam');
```

Comment:
it appears, that viewing `kubra_dam` at http://localhost:8081/ASP/mapinfo/view/0 does not work because its `id == 0` (https://github.com/startersclan/ASP/commit/1aae286ec5566cb3acedfeffb112284cee79f1f7 ), where it's coded to redirect back to http://localhost:8081/ASP/mapinfo. 

The schema of the `map` table does not constrain the `id` column to be `> 0`, so I guess  `0` is a perfectly valid value. I guess that `Mapinfo.php` bug should be fixed in another PR. What do you think? Did i miss something, like perhaps `0` was for some other purpose?

Reply:
Doing a [search of all occurrences of `id == 0`](https://github.com/search?q=repo%3Astartersclan%2Fasp+%22if+%28%24id+%3D%3D+0%29%22&type=code) shows that `0` is somehow assume to be an invalid value. I guess should this be changed to another value?

Reply:
This might be what Wilson has previously spoken of in the patch notes and the forums? (pretty sure I read that somewhere) We could certainly change it as a fix, it'd mean changing the constants.py on the client side too.

I'll think about for a little bit and see if I can come up with a solution.

Reply:
Easy enough fix is just to change that to id < 0

Reply:
Oh so it corresponds to `constants.py` on the client side. My knowledge of this is limited 😄 Let me try to wrap my head around it.

Reply:
Gotcha i see it in `./python/bf2/stats/constants.py`. Yep I think we'll implement your idea. Thank you 😃

Reply:
Ok lets merge this PR, need you to force push only the first commit on this PR, and open another PR with the fix commit 😃

Reply:
Is there an easier way to do this other than creating a new PR?

Reply:
not that i know, for simultaneous open PRs, each must possess its own branch. 😄

Reply:
> Oh so it corresponds to `constants.py` on the client side. My knowledge of this is limited 😄 Let me try to wrap my head around it.

Basically, the client side python files bundle up a round log in a json file that gets sent to the server. More specifically, it checks bf2statistics.php at the server address you specifiy in BF2StatisticsConfig.py, then bf2statistics.php processes the incoming json and generates a copy in one of the on server directories, 'unauthorized', 'failed', 'processed' or 'unprocessed'

Reply:
Haha, yes would be nice to have time to study these more. I did spent some time fixing the python scripts before, so i understand most of how the snapshot is sent, didn't know how it worked with `constants.py`, but now I know 😄 Hopefully we get somewhere with this 😶‍🌫️ 

It's cool when the entire things works nicely 👍